test/testOutput.cpp

```diff
@@ -251,6 +251,25 @@ TEST(ParticleCollector, getTrajectory) {
 	EXPECT_TRUE(ArraysMatch(pos_x_expected, pos_x));
 }
 
+TEST(ParticleCollector, runModuleList) {
+        ModuleList modules;
+        modules.add(new SimplePropagation());
+        modules.add(new MaximumTrajectoryLength(1 * Mpc));
+        
+        ParticleState p;
+        p.setPosition(Vector3d(10, 0, 0));
+        p.setDirection(Vector3d(-1, 0, 0));
+        ref_ptr<Candidate> c = new Candidate(p);
+
+        ref_ptr<ParticleCollector> collector = new ParticleCollector();
+
+        collector->process(c);
+        
+        modules.setShowProgress(false);
+        auto candidates = collector->getContainer();
+	modules.run(&candidates);
```

Comment:
Minor comment: identation on the last line.

Reply:
OK, thanks for spotting. I've corrected it consistently with clang-format.